src/client/uimap.js

```diff
@@ -13,6 +14,45 @@ UIMap.initialize = async function() {
   const basic = await WZManager.get('UI.wz/Basic.img');
   this.statusBarLevelDigits = basic.LevelNo.nChildren.map(d => d.nGetImage());
 
+  this.chat = new MapleInput({
+    x: 5,
+    y: 540,
+    width: 530,
+    color: '#000000',
+    background: '#ffffff',
+    height: 13
+  });
+  this.chat.input.style.zIndex = 4;
+  this.chat.addSubmitListener(() => {
+    const msg = this.chat.input.value;
+    this.chat.input.value = "";
+    if (msg[0] == '!') {
+      const command = msg.split(' ');
+      console.log(command);
+      switch (command[0]) {
+        case "!level":
```

Comment:
I recommend wrapping the case bodies inside braces. See https://github.com/airbnb/javascript#comparison--switch-blocks.